### DISCUSSION TITLE
Kapp(helm) installation fails on AWS EKS

### DISCUSSION THREAD

**Discussion by haghayeghh** [wundergraph/cosmo#870] [General]
The helm installation fails on EKS with below error on cosmo-controlplane-seed-organization:

`seed
response: Response {
seed
status: 403,
seed
statusText: 'Forbidden',
seed
headers: Headers {
seed
'referrer-policy': 'no-referrer',
seed
'x-frame-options': 'SAMEORIGIN',
seed
'strict-transport-security': 'max-age=31536000; includeSubDomains',
seed
'cache-control': 'no-store',
seed
'x-content-type-options': 'nosniff',
seed
pragma: 'no-cache',
seed
'x-xss-protection': '1; mode=block',
seed
'content-type': 'application/json',
seed
'content-length': '64'
seed
},
seed
body: ReadableStream { locked: true, state: 'closed', supportsBYOB: true },
seed
bodyUsed: true,
seed
ok: false,
seed
redirected: false,
seed
type: 'basic',
seed
url: 'http://cosmo-keycloak:8080/realms/master/protocol/openid-connect/token'
seed
},
seed
responseData: { error: 'invalid_request', error_description: 'HTTPS required' }
seed
}`

also the studio UI is loaded with this error 
`
Could not retrieve federated graphs
[unknown] To use Connect, you must provide a `Transport`: a simple object that handles `unary` and `stream` requests. `Transport` objects can easily be created by using `@connectrpc/connect-web`'s exports `createConnectTransport` and `createGrpcWebTransport`. see: https://connectrpc.com/docs/web/getting-started for more info.
`

**Comment by StarpTech** [comment]
Hi, please try again with the latest release and feel free to reopen in case of reproduction.
